packages/theming/theme-tokens/src/getTokens.ios.ts

```diff
@@ -1,9 +1,24 @@
+import iOSLightAliasTokens from '@fluentui-react-native/design-tokens-ios/light/tokens-aliases.json';
+import iOSDarkAliasTokens from '@fluentui-react-native/design-tokens-ios/dark/tokens-aliases.json';
 import iOSLightShadowTokens from '@fluentui-react-native/design-tokens-ios/light/tokens-shadow.json';
 import iOSDarkShadowTokens from '@fluentui-react-native/design-tokens-ios/dark/tokens-shadow.json';
+
 import { AppearanceOptions } from '@fluentui-react-native/theme-types';
 import { assertNever } from 'assert-never';
 
-export function getiOSShadowTokens(mode: AppearanceOptions) {
+export function getAliasTokens(mode: AppearanceOptions) {
+  if (mode === 'light') {
+    return iOSLightAliasTokens;
+  } else if (mode === 'dark') {
+    return iOSDarkAliasTokens;
+  } else if (mode === 'highContrast') {
```

Comment:
Curious - how did you hot this assert at runtime flows from the tester?

Reply:
The default theme was always getting created on startup for iOS - actually, all three types of default theme (light mode, dark mode, and high contrast mode) were getting created (https://github.com/microsoft/fluentui-react-native/blob/main/packages/theming/default-theme/src/createDefaultTheme.ts), regardless of what mode iOS was on. After the getiOSTokens.ts to getTokens.ios.ts rename, the default theme started using iOS tokens to get created rather than web/win32 tokens. So we were trying to create a high contrast default theme using iOS tokens, which doesn't define any. 

Android should be hitting this too, but rather than throwing an error, Android just returns the light mode tokens. I might make the same change for iOS, until we can come up with an overall plan for deciding what logic each theme should contain and how the default theme should work